pjlib/build/Makefile

```diff
@@ -52,7 +52,7 @@ export TEST_OBJS += activesock.o atomic.o echo_clt.o errno.o exception.o \
 		    select.o sleep.o sock.o sock_perf.o ssl_sock.o \
 		    string.o test.o thread.o timer.o timestamp.o \
 		    udp_echo_srv_sync.o udp_echo_srv_ioqueue.o \
-		    util.o
+		    util.o socketpair.o
```

Comment:
I would suggest putting the test in existing `sock.c` to simplify changes.

Reply:
Any reason why this is not done?

Reply:
sorry forgot it. I have changed it.